README.md

```diff
@@ -5,25 +5,31 @@
 Europeana Portal with Collections as a Rails + 
 [Blacklight](https://github.com/projectblacklight/blacklight) application.
 
-## License
-
-Licensed under the EUPL V.1.1.
-
-For full details, see [LICENSE.md](LICENSE.md).
-
 ## Requirements
 
-* Ruby 2 (latest stable version recommended)
+* Ruby 2.5
+* PostgreSQL 9.4+
+* Redis
+* S3-compatible object storage
 * ImageMagick
 * A key for the Europeana REST API, available from:
   http://labs.europeana.eu/api/registration/
-* PostgreSQL 9.4+
 
 ## Installation
 
 * Download the source code
 * Run `bundle install`
 
+## Quick Start with Docker
+
+* Install the gem bundle: `bundle install`
+* Generate Docker configuration files: `bundle exec bin/portal dockerize development`
+  You will be prompted to enter your Europeana API key.
+* Bring it up: `docker-compose -f docker-compose.services.yml up`
+* Setup the database: `bundle exec rake db:setup`
```

Comment:
either here or down below in the test section add instructions for setting up the test db.
All I needed to do was after the above commands, copy .env.development to .env.test and change the DB name. Then run: 'RAILS_ENV=test bundle exec rake db:setup'

The copying of the .env file is something the dockerize script could already do perhaps?